crates/biome_grit_patterns/tests/specs/ts/import_patterns.snap

```diff
@@ -0,0 +1,13 @@
+---
+source: crates/biome_grit_patterns/tests/spec_tests.rs
+expression: import_patterns
+---
+SnapshotResult {
+    messages: [],
+    matched_ranges: [
+        "2:1-2:27",
+        "2:1-2:27",
```

Comment:
I am slightly confused why there are two matches at this position. I think there should be only one?